src/index.ts

```diff
@@ -2521,6 +2521,765 @@ export enum ButtonVariant {
+  list: PurposeListThemeConfig
+  footer: FooterThemeConfig
+}
+
+/** Preferences - Subscriptions Theme Config */
+
+export enum SubscriptionListStyle {
+  Filled = 'filled',
+  Underlined = 'underlined',
+}
+
+export interface SubscriptionsTabGlobalControlThemeConfig {
+  background: ColorThemeConfig
+  text: ColorThemeConfig
+  cornerRadius: number
+  switchButton: SwitchButtonsThemeConfig
+}
+
+export interface LayoutThemeConfig {
+  style: SubscriptionListStyle
+  background: ColorThemeConfig
+  text: TextThemeConfig
+  link: TextThemeConfig
+  cornerRadius: number
+}
+
+export interface SubscriptionsTabListThemeConfig {
+  layout: LayoutThemeConfig
+  switchButton: SwitchButtonsThemeConfig
+  checkbox: CheckboxesThemeConfig
+}
+
+export interface PreferenceSubscriptionsTabThemeConfig {
+  header: PageTextThemeConfig
+  globalControl: SubscriptionsTabGlobalControlThemeConfig
+  list: SubscriptionsTabListThemeConfig
+  footer: FooterThemeConfig
+}
+
+/** Preferences - Requests Theme Config */
+
+export enum FormFieldStyle {
+  Box = 'box',
+  Underline = 'underline'
+}
+
+export interface RequestsTabDsrLinkThemeConfig {
+  title: TextThemeConfig
+  description: TextThemeConfig
+  arrowIcon: ColorThemeConfig
+  background: ColorThemeConfig
+  cornerRadius: number
+}
+
+export interface RightsListItemThemeConfig {
+  title: TextThemeConfig
+  subtitle: TextThemeConfig
+  arrowIcon: ColorThemeConfig
+  background: ColorThemeConfig
+  cornerRadius: number
+}
+
+export interface RequestsTabRightsListThemeConfig {
+  title: TextThemeConfig
+  item: RightsListItemThemeConfig
+}
+
+export interface RequestsTabHomeThemeConfig {
+  header: PageTextThemeConfig
+  dsrPortalLink: RequestsTabDsrLinkThemeConfig
+  rightsList: RequestsTabRightsListThemeConfig
+}
+
+export interface FormDividersThemeConfig {
+  title: TextThemeConfig
+  subtitle: TextThemeConfig
+}
+
+export interface FieldThemeConfig {
+  background: ColorThemeConfig
+  outline: ColorThemeConfig
+}
+
+export interface FormFieldsThemeConfig {
+  style: FormFieldStyle
+  active: FieldThemeConfig
+  inactive: FieldThemeConfig
+  cornerRadius: number
+  fieldLabel: TextThemeConfig
+  hintText: TextThemeConfig
+  inputText: TextThemeConfig
+}
+
+export interface RequestFormFormThemeConfig {
+  dividers: FormDividersThemeConfig
+  fields: FormFieldsThemeConfig
+  checkboxes: CheckboxesThemeConfig
+  actionButton: ActionButtonThemeConfig
+}
+
+export interface RequestsTabRequestFormThemeConfig {
+  header: HeaderThemeConfig
+  form: RequestFormFormThemeConfig
+}
+
+export interface SubmittedRequestThemeConfig {
+  bannerVisible: boolean
+  bannerImage: ImageThemeConfig
+  title: TextThemeConfig
+  description: TextThemeConfig
+}
+
+export interface RequestsTabSubmittedThemeConfig {
+  header: HeaderThemeConfig
+  summary: SubmittedRequestThemeConfig
+}
+
+export interface PreferenceRequestsTabThemeConfig {
+  home: RequestsTabHomeThemeConfig
+  requestForm: RequestsTabRequestFormThemeConfig
```

Comment:
What to show?

Reply:
Example form or grey box?

Reply:
For Misha meeting:
- Should we consider showing just a grey box in exp config, because form isn't configured here
- Should we make the values that are being configured in formTheme more explicit

Reply:
Example form it is